### DISCUSSION TITLE
Pin Last column and first column is not working on rtl direction

### DISCUSSION THREAD

**Discussion by samimhamza** [icflorescu/mantine-datatable#551] [Q&A]
This is my DataTable code everything is working fine but pin last column and first column is not working on rtl direction, it work or ltr direction.

`<DataTable
			height={550}
			withTableBorder
			highlightOnHover
			withColumnBorders
			striped
			verticalAlign="top"
			pinLastColumn
			columns={columns}
			fetching={isLoading}
			records={data?.data}
			page={data?.page ? data?.page : 1}
			onPageChange={handlePageChange}
			totalRecords={data?.total}
			recordsPerPage={data?.per_page ? data?.per_page : 20}
			sortStatus={sortStatus}
			onSortStatusChange={handleSortStatusChange}
			selectedRecords={selectedRecords}
			onSelectedRecordsChange={setSelectedRecords}
			shadow="sm"
			borderRadius="sm"
			loadingText={t("loading_data")}
			noRecordsText={t("no_records")}`

**Comment by icflorescu** [comment]
Open-source projects are created, maintained and made better by people like [Reem Assaf](https://github.com/ReemX), [whose contribution](https://github.com/icflorescu/mantine-datatable/pull/781) finally added RTL support to [Mantine DataTable](https://icflorescu.github.io/mantine-datatable/)! 🎉🙏
